src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UpdatePreviewResult.cs

```diff
@@ -9,18 +11,24 @@ public class UpdatePreviewResult
     {
         public PackageIdentity Old { get; }
         public PackageIdentity New { get; }
+        private readonly string _automationName;
```

Comment:
Why? 
That's the benefit? 
Just use a readonly property.

Reply:
I wasn't aware of c# read only properties. Thanks!